text/0000-language-semver.md

```diff
@@ -0,0 +1,425 @@
+- Feature Name: N/A
```

Comment:
Is this file meant to be included? It seems like an early version of the other one.